assets/scripts/build_gallery.py

```diff
@@ -596,8 +750,9 @@ def _disable_progress_bars() -> Tuple[str, DataContext]:
     "--outfile-name",
     "-o",
     "outfile_name",
-    default="expectation_library_v2--staging.json",
-    help="Name for the generated JSON file",
+    # default="expectation_library_v2--staging.json",
+    default="expectation_library_v2--PLAY.json",
```

Comment:
Should this be reverted or is this intentional?

Reply:
Intentional. In the PR description, there is a **BEFORE MERGE** checklist that is to revert this. Just don't want to accidentally overwrite anything.